scripts/build.mjs

```diff
@@ -33,11 +33,15 @@ await $`emcmake cmake -G Ninja ..`;
 await $`cmake --build . -j ${jobs}`;
 
 // Copy artifacts to dist
-await fs.copy("postject.wasm", "../dist/postject.wasm");
 await fs.copy("postject.js", "../dist/postject.js");
-await fs.copy("../src/main.js", "../dist/main.js");
+await fs.copy("../src/api.js", "../dist/api.js");
+await fs.copy("../src/cli.js", "../dist/cli.js");
 await fs.copy("../postject-api.h", "../dist/postject-api.h");
 
+// Bundle ../dist/postject.js into ../dist/api.js
+await $`esbuild ../dist/api.js --allow-overwrite --bundle --platform=node --outfile=../dist/api.js`;
+await fs.rm("../dist/postject.js");
```

Comment:
Probably simpler to copy `../src/api.js` into `build` and then bundle? Rather than copying to `dist` then overwriting it, and then you also don't have to copy `postject.js` to `dist`.

Reply:
@dsanders11 done, PTAL